### PR TITLE
Fixing heartbeat job stale alert

### PR DESCRIPTION
**Pull request by chaodaiG**
the previous absent_for condition was not applicable because the result was gauge value

### CODE REVIEW

**Comment by fejta** [comment]
/uncc @chases2 @fejta 
Add me back if you want me to review!

File: prow/oss/terraform/modules/alerts/main.tf [hunk 1]
```diff
@@ -115,13 +115,20 @@ resource "google_monitoring_alert_policy" "heartbeat-job-stale" {
...
     display_name = "heartbeat-job-stale"
 
-    condition_absent {
-      filter   = "metric.type=\"workload.googleapis.com/prowjob_state_transitions\" resource.type=\"k8s_container\" metric.label.\"job_name\"=\"${var.heartbeat_job.job_name}\" metric.label.\"state\"=\"success\""
+    condition_monitoring_query_language {
       duration = "${var.heartbeat_job.alert_interval}"
-
-      aggregations { # required
-        alignment_period     = "${var.heartbeat_job.interval}"
-        per_series_aligner   = "ALIGN_RATE"
```
Comment (cjwagner) [inline prow/oss/terraform/modules/alerts/main.tf:124]: I think switching this to `ALIGN_DELTA` would also be equivalent.

**Review by cjwagner — APPROVED** [review]
LGTM, either way works.
/hold

File: prow/oss/terraform/modules/alerts/main.tf (hunk 1 above)
Comment (chaodaiG) [reply prow/oss/terraform/modules/alerts/main.tf:124]: I have tested it, it didn't work as I was expecting

**Pull request merged**